src/test/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactoryTest.java

```diff
@@ -578,4 +578,13 @@ public void testVariantIds() {
         assertEquals(expResult, result);
         assertEquals(emptySet, result.get(0).getIds());
     }
+
+    @Test
+    public void testIgnoreRecordsWithNonVariantGenotypes() {
+        String line = "1\t10040\trs123\tT\tC\t.\t.\t.\tGT\t0/0\t0/1\t0/.\t./1\t1/1"; // 5 samples, 1 with non-variant genotype
```

Comment:
this should be using only "0/0", "0|0", and "./.". see the comment on the actual requirements